CMakeLists.txt

```diff
@@ -121,7 +121,9 @@ add_executable(${EXE_NAME}
 		include/vk/cmdSync.h
 		src/vk/cmdSync.cpp
 		include/app/config/JSON.h
-		include/app/log.hpp)
+		include/app/log.hpp
+		src/app/log.cpp
+)
```

Comment:
`clock.h` and `clock.cpp` are missing

Reply:
clock.h and clock.cpp had been a part of the main target even before I added timestamps